src/pkg/cli/client/byoc/aws/validation.go

```diff
@@ -73,28 +73,19 @@ func ValidateGPUResources(ctx context.Context, project *composeTypes.Project) er
 	// return after checking if there are actually non-zero GPUs requested
 	hasGPUs, quotaErr := hasGPUQuota(ctx)
```

Comment:
can we rename this to `gpusAvailable` or something like that to differentiate it from the notion of whether or not project has GPUs?